pkg/security/probe/probe_bpf.go

```diff
@@ -295,6 +297,12 @@ func (p *Probe) handleEvent(data []byte) {
 	}
 	offset += read
 
+	// check event order
+	if event.TimestampRaw < p.lastTimestamp && p.lastTimestamp != 0 {
+		_ = p.statsdClient.Count(MetricPrefix+".event.reorder_error", 1, []string{}, 1.0)
```

Comment:
Nit: to follow the pattern of the other metrics, you might want to call it `.events.`. I would also call it `differed` or `sorting_error` just to make sure we know what kind of error this is ... but that's really just a nit :)

Reply:
+1 for the s/error/errors/